content/en/docs/guides/opentelemetry-metrics.md

```diff
@@ -0,0 +1,113 @@
+---
+title: OpenTelemetry Metrics
+description: >-
+  OpenTelemetry Metrics available in gRPC
+---
+
+### Overview
+
+gRPC provides in-built support for an OpenTelemetry plugin that provides metrics that can help you troubleshoot your system, iterate on improving system performance and setup continuous monitoring and alerting.
```

Comment:
I've always seen this as "built-in", do you see that being used as well?

Reply:
actually, both in-built or built-in sound wrong, since they are plugins on top of gRPC. I'm gonna remove this